index.html

```diff
@@ -140,10 +219,47 @@ <h2 class="tonic">Tonic</h2>
                </ul>
                <button class="see-project">See Project</button>
            </div>
+           </div>
+            </section>
+            <section class="sec4 desktop">
+              <div class="sec-content">
+                <div class="image-div"> 
+                  <img src="images/tonic-desk.jpg" alt="image" class="workplace-image">
+                </div>
+                <div class="sec-cont">
+                <h2 class="tonic desk">Facebook 360</h2>
+                <ul class="set">
+                  <li>
+                    <p class="canopy">FACEBOOK</p> 
+                  </li>
+                  <li class="dot-style">
+                        <img width="10px" height="10px" src="images/Dot.png" alt="dot">
+                       <p class="back">Full Stack Dev</p>
+                      </li>
+                      <li class="dot-style">
+                        <img width="10px" height="10px" src="images/Dot.png" alt="dot">
+                       <p class="year">2015</p>
+                      </li>
+                   </ul>
+                   <p class="daily">
+                    Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.
+                   </p>
+                   <ul class="list-tech f-margin">
+                       <li>html</li>
+                       <li>Ruby on rails</li>
+                       <li>css</li>
+                       <li>javascript</li>
+                   </ul>
+                   <button class="see-project blue b-margin">See Project</button>
+               </div>
+               </div>
             </section>
```

Comment:
- [OPTIONAL] I have noticed that you are creating a new desktop card for the desktop but the main purpose of using media queries is to adapt our website to different screens by creating different styles for the same Html file without duplicating it. I highly recommend you not create a new project card for the desktop but to use CSS to manipulate styles and how it must appear on the different screens without changing your Html file entirely. :wink: Please to d all that on all project cards.